src/main/scala/ingest/PyramidTobler.scala

```diff
@@ -85,29 +109,89 @@ object ToblerPyramid {
             }
         }
 
+      /* ORC file is assumed to be a snapshot with the most recent versions of every Element. */
+      osm.fromORC(orc) match {
+        case Left(err) => println(err)
+        case Right((ns, ws, _)) => {
+          val roads: RDD[(Long, osm.Way)] = ws.filter { case (_, w) => w.meta.tags.contains("highway") }
+
+          val lines: RDD[Line] = osm.toHistory(ns, roads)._2.map(_.geom)
```

Comment:
For the moment, `toHistory` has the more convenient return type. Luckily it won't do extra work even if the ORC file given to it wasn't a history file.